aspnet/publishing/web-publishing-vs.rst

```diff
@@ -21,11 +21,11 @@ The following image shows a visualization of this process.
 
 In the image above each black circle ● indicates an extension point, we will cover each extension point later in this document.
 
-When you start a publish operation, the publish dialog is closed and then MSBuild is called to start the process. Visual Studio calls MSBuild to do this so that you can have parity with publishing when using Visual Studio or the command line. The MSBuild layer is pretty thin, for the most part it just calls dnu publish. Let’s take a closer look at dnu publish.
+When you start a publish operation, the publish dialog is closed and then MSBuild is called to start the process. Visual Studio calls MSBuild to do this so that you can have parity with publishing when using Visual Studio or the command line. The MSBuild layer is pretty thin, for the most part it just calls ``dotnet publish``. Let’s take a closer look at ``dotnet publish``.
```

Comment:
We augment "dotnet publish" instead of creating a separate "publish-iis" command that the user calls.